scripts/coins-logon-widget.js

```diff
@@ -1,115 +1,79 @@
 'use strict';
-var EventEmitter = require('wolfy87-eventemitter');
-var assign = require('lodash/object/assign');
-var cloneDeep = require('lodash/lang/cloneDeep');
+
 var auth = require('./lib/auth');
-var Form = require('./lib/form');
-var FormGroup = require('./lib/form-group');
+var bind = require('lodash/function/bind');
+var createElement = require('virtual-dom/create-element');
+var diff = require('virtual-dom/diff');
+var EventEmitter = require('wolfy87-eventemitter');
```

Comment:
It's a browser-ready event emitter library which `CoinsLogonWidget` extends. We could just bundle Node's `EventEmitter`, too.